Plots/Contours/NCL_ce_1.py

```diff
@@ -0,0 +1,95 @@
+"""
+NCL_ce_1.py
+===========
+This script illustrates the following concepts:
+   - Drawing black-and-white contours over a cylindrical equi-distant map
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/ce_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/ce_1_lg.png
+"""
+
+###############################################################################
+# Import packages:
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+from cartopy.mpl.gridliner import LongitudeFormatter, LatitudeFormatter
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a netCDF data file using xarray default engine and load the data into xarrays, choosing the 2nd timestamp
+ds = xr.open_dataset(gdf.get("netcdf_files/uv300.nc")).isel(time=1)
+
+###############################################################################
+# Plot:
+
+# Generate figure (set its size (width, height) in inches)
+fig = plt.figure(figsize=(12, 6))
+
+# Generate axes using Cartopy projection
+projection = ccrs.PlateCarree()
+ax = plt.axes(projection=projection)
+
+# Draw land
+ax.add_feature(cfeature.LAND, color='silver')
+
+# Define the contour levels
+levels = np.arange(-12, 41, 4)
+
+# Draw contour lines
+contour = ds.U.plot.contour(
+    ax=ax,
+    levels=levels,
+    xticks=np.arange(-180, 181, 30),  # nice x ticks
+    yticks=np.arange(-90, 91, 30),  # nice y ticks
+    transform=projection,  # ds projection
+    add_labels=False,  # turn off xarray's automatic Lat, lon labels
+    colors="black",  # note plurals in this and following kwargs
+    linestyles="-",
+    linewidths=0.5)
+
+# Label the contours and set axes title
+ax.clabel(contour, levels, fontsize=12, fmt="%.0f")
```

Comment:
```suggestion
ax.clabel(contour, [-8, 0, 8, 16, 24, 32], fontsize=12, fmt="%.0f")
```
So this is super annoying because of how the NCL version did labels vs what it describes as the levels and so fourth. The contour levels are indeed -12 to 40 by 4, but the actual labels are as I have specified in this change.

Reply:
Ah I see, I did not realize this. Thank you!